pixart.js

```diff
@@ -1 +1,37 @@
+var button = $('#set-color')
+var brush = $('.brush')
+var colorField = $('#color-field')
+var square = $('.square')
+var first = $('#first')
+var second = $('#second')
+var third = $('#third')
+var holderColor = null
 
+button.on('click',function(){
+  setColor();
+  return false;
+})
+colorField.on('keypress', function(e){
+    if (e.keyCode == 13) {
+      setColor();
+    }
+});
+$('.history').children().on('click',function(){
+  holderColor = $(this).css('background')
+  $(this).css('background',brush.css('background'))
+  brush.css('background',holderColor)
+})
+
```

Comment:
I will keep this in mind for the project. In fact, I've already made sure to not do this and made a comment about it haha.  Thanks